Android/app/src/main/AndroidManifest.xml

```diff
@@ -29,6 +28,24 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity
```

Comment:
We should only have 2 activities, MainAppActivity and LoginAndSignUpActivity. The rest are fragments connected by the navigation graphs.